loadtest/scripts/populate_genesis_accounts.py

```diff
@@ -46,33 +41,14 @@ def add_account(account_name, address, mnemonic, local=False):
         }
         json.dump(data, f)
 
-    return add_account_cmd
-
 
 def create_genesis_account(account_index, account_name, local=False):
     address, mnemonic = add_key(account_name=account_name, local=local)
-    add_account_cmd = add_account(account_name=account_name, address=address, mnemonic=mnemonic, local=local)
-
-    retry_counter = 0
-    sleep_time = 1
-
-    while True and retry_counter < 1000:
-        try:
-            print(f'Running: ${add_account_cmd}')
-            subprocess.call(
-                [add_account_cmd],
-                shell=True,
-                timeout=20,
-            )
-            break
-        except subprocess.CalledProcessError as e:
-            print(f"Encountered error {e}, retried {retry_counter} times")
-            retry_counter += 1
-            sleep_time += 0.5
-            time.sleep(sleep_time)
-
-    if retry_counter >= 1000:
-        exit(-1)
```

Comment:
We're already calling add keys and manually adding a balance entry in the bank module so we don't actually need to call this. Its just a wrapper